release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -817,6 +817,11 @@ Zero touch provisioning (ZTP) uses the Metal3 service to boot RHCOS on the targe
 
 For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc#about-ztp_ztp-deploying-far-edge-clusters-at-scale[Using ZTP to provision clusters at the network far edge].
 
+[id="ocp-4-12-hcp-node-tuning-operator"]
+==== Node tuning in a hosted cluster (Technology Preview)
+
+You can now configure node tuning in a hosted cluster. To configure node tuning, you use the Node Tuning Operator to create config maps that contain `Tuned` objects and reference the config maps in your node pools. For advanced node tuning, you also use the Node Tuning Operator and set kernel boot parameters. For more information, see xref:../scalability_and_performance/using-node-tuning-operator.adoc#node-tuning-hosted-cluster_node-tuning-operator[Configuring node tuning in a hosted cluster].
```

Comment:
Thanks Laura for the PR.

There are a few inaccuracies that I would like to address. How does this sound to you?
```suggestion
You can now configure OS-level tuning for nodes in a hosted cluster using the Node Tuning Operator. To configure node tuning, you can create config maps in the management cluster that contain `Tuned` objects, and reference these config maps in your node pools. The tuning configuration defined in the `Tuned` objects will be applied to the nodes in the node pool. For more information, see xref:../scalability_and_performance/using-node-tuning-operator.adoc#node-tuning-hosted-cluster_node-tuning-operator[Configuring node tuning in a hosted cluster].
```

Reply:
@dagrayvid You should be able to see the latest updates now.